yarn.lock

```diff
@@ -15,31 +15,6 @@ __metadata:
   languageName: node
```

Comment:
Can you also delete `.yarn` directory, `.yarnrc.yml` and `.yarn` references in the gitignore files?

Reply:
We are still using Yarn. see comment above.